DigitalOffice/Services/ProjectService/Controllers/ProjectController.cs

```diff
@@ -1,4 +1,28 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+<<<<<<< HEAD
```

Comment:
Resolve conflict. Write to me if you need help with this, I'll show you how it's done.